### DISCUSSION TITLE
是否有想法做成员个人博客的汇总（引流

### DISCUSSION THREAD

**Discussion by Augists** [NAOSI-DLUT/.github#3] [Polls]
这是一个投票贴
**鼓励大家搭建自己的个人博客**，可以参考如hexo, hugo, vitepress, wordpress等框架，也可以向方神学习自己写前后端做博客
**鼓励大家博客建立友链**，你可以直接在下方添加你个人博客的信息，一般来说需要提供
* Nickname
* Url
* Avatar (as url)
* Simple Description

如有额外需要联系楼主修改添加

如果有一些支持的人，会建立仓库以PR的形式由大家参与构建汇总

**Comment by li-ruida** [comment]
强烈推荐[halo](https://halo.run/)
